### DISCUSSION TITLE
How to add custom images in pnp search vertical

### DISCUSSION THREAD

**Discussion by Arkadip89** [microsoft-search/pnp-modern-search#1991] [Q&A]
Hello team,
We are working on PnP - Search Verticals , we have one requirement that is- rather adding Fluent UI icon we want to add our custom images/icon in the PnP - Search Vertical. is it possible? I have tried a lot but not able to do it.

If it is possible please give me some hint so that I can proceed further. 

Thank you in advance. Please help me if possible. Please refer the attached images.

Regards,
Arkadip Ghsoh
![Untitled](https://user-images.githubusercontent.com/102343888/161194235-14fac1d2-1620-4745-9910-5c41bdd5ae8d.png)

**Comment by wobba** [comment]
Currently not possible.
